pandas/_libs/interval.pyx

```diff
@@ -423,7 +423,14 @@ cdef class Interval(IntervalMixin):
             msg = '`other` must be an Interval, got {other}'
             raise TypeError(msg.format(other=type(other).__name__))
 
+        def is_empty(intv):
+            return intv.left==intv.right and intv.closed != 'both'
```

Comment:
small nitpick: add a space around each side of the `==` (I don't think our linters are super strict on the `.pyx` files)

Reply:
fixed.